moe.clover.mm3d.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: moe.clover.mm3d
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: mm3d
+finish-args:
+  # OpenGL access
+  - --device=dri
+  # Wayland access
+  - --socket=wayland
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # File access
+  - --filesystem=host
```

Comment:
This seems too broad? Does it use filechooser/filechooser portal? Which paths does it need?

Reply:
I wrote a long response so I posted in the main comments at https://github.com/flathub/flathub/pull/4833#issuecomment-1878058690.